sbin/build-push-docker-images.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This script builds and pushes docker images when run from a release of Spark
+# with Kubernetes support.
+
+declare -A path=( [spark-driver]=dockerfiles/driver/Dockerfile \
+                  [spark-executor]=dockerfiles/executor/Dockerfile \
```

Comment:
Execution failed
```
./build-push-docker-images.sh: line 22: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /Users/jimmy/Workspace/github/rootsongjc/spark/dist/sbin/dockerfiles: no such file or directory
unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /Users/jimmy/Workspace/github/rootsongjc/spark/dist/sbin/dockerfiles: no such file or directory
```

Reply:
I think that's because of an older bash version. https://stackoverflow.com/questions/6047648/bash-4-associative-arrays-error-declare-a-invalid-option

Reply:
MacOS bash version
```
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin16)
Copyright (C) 2007 Free Software Foundation, Inc.
```
I updated it with this command:
```
brew install bash
```
Now bash version is 4.4.12(1)-release (x86_64-apple-darwin16.3.0)
The error disappeared.

Reply:
As this script located at `sbin/build-push-docker-images.sh` but the path in the script based on `dockerfile`, this directory not in the base path.

Reply:
It is meant to be run from a release that is built and published.

Reply:
I put it to `dist` directory and run the scripts.